globus_automate_client/cli/callbacks.py

```diff
@@ -155,3 +198,29 @@ def flows_endpoint_envvar_callback(default_value: str) -> str:
     defining the target Flow endpoint.
     """
     return os.getenv("GLOBUS_AUTOMATE_FLOWS_ENDPOINT", default_value)
+
+
+def input_validator_callback(body: str) -> str:
```

Comment:
Still getting through the whole thing, so this comment may become moot or more clear, but seems redundant with the file loading stuff in the yaml validator.

Perhaps I'll discover this structure going forward, but it does seem like doing a layer first of determining whether the parameter loads as a file first and returning that file content or the original string, followed by doing appropriate json/yaml parsing is a good approach. Thus, we don't have file open/read in multiple places.

I guess doing so means that all the json/yaml parsing will be string rather than file based, but assuming that doesn't mean we lose information that would be helpful in error reporting, that seems ok to me.